### DISCUSSION TITLE
Skip_TPM_Check_on_Dynamic_Update.cmd, Should I rerun after upgrade?

### DISCUSSION THREAD

**Discussion by gurumagic** [AveYo/MediaCreationTool.bat#323] [General]
I am using the Skip_TPM_Check_on_Dynamic_Update.cmd script. 

When I launch it, it says to "rerun" the script to remove. Is it best practice to rerun the script after a successful upgrade?

**Comment by nondetect** [comment]
If you want to receive update in future, leave it as is

**Reply by gurumagic** [reply]
Got it.  Thanks
